src/main/java/de/jcm/discordgamesdk/impl/events/ActivityJoinEvent.java

```diff
@@ -0,0 +1,31 @@
+package de.jcm.discordgamesdk.impl.events;
+
+import de.jcm.discordgamesdk.Core;
+import de.jcm.discordgamesdk.LogLevel;
+import de.jcm.discordgamesdk.impl.Command;
+import de.jcm.discordgamesdk.impl.EventHandler;
+import de.jcm.discordgamesdk.user.DiscordUser;
+
+public class ActivityJoinEvent {
+    public static class Handler extends EventHandler<DiscordUser>
+    {
+        public Handler(Core.CorePrivate core)
+        {
+            super(core);
+        }
+
+        @Override
+        public void handle(Command command, DiscordUser user)
+        {
+            core.log(LogLevel.VERBOSE, "Secret: " + command.getData().getAsJsonObject().get("secret").getAsString());
```

Comment:
Do we need to log this?

Reply:
oh i forgot to remove that when trying to add the feature, sorry, will push a commit removing that later